modules/ROOT/pages/design-delete-project.adoc

```diff
@@ -0,0 +1,23 @@
+= Delete a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can delete an API-specification project or an API-fragment project.
+
+[NOTE]
+====
+If you have already published a project, deleting it in API Designer does not delete it from Anypoint Exchange.
+====
+
+[IMPORTANT]
+====
+The deletion of a project is final. Deleting a project does _not_ move it to a *Trash* folder from which you can restore the project if you change you mind.
+====
+
```

Comment:
One container topic followed by just one subtopic?
I was taught that you need at least two subtopics to break into another level.
But I suppose you are following your own internal style of always having a Procedure topic?
This one is so short that it could easily be Delete a Project, with no notices or subtitle.

Reply:
@kbostrom , generally, yes. Each section should have more than one subsection, if it is to have subsections. However, context allows for exceptions.